datashuttle/utils/rclone.py

```diff
@@ -42,28 +41,33 @@ def transfer_data(
 
     if upload_or_download == "upload":
 
-        call_rclone(
+        output = call_rclone(
             f"{rclone_args('copy')} "
             f'"{local_filepath}" '
             f'"{rclone_config_name}:'
             f'{remote_filepath}" '
-            f"{extra_arguments}"
+            f"{extra_arguments}",
+            pipe_std=True,
         )
 
     elif upload_or_download == "download":
-        call_rclone(
+        output = call_rclone(
             f"{rclone_args('copy')} "
             f'"{rclone_config_name}:'
             f'{remote_filepath}" '
             f'"{local_filepath}"  '
-            f"{extra_arguments}"
+            f"{extra_arguments}",
+            pipe_std=True,
         )
 
+    return output
+
 
 def setup_remote_as_rclone_target(
     cfg: Configs,
     rclone_config_name: str,
     ssh_key_path: Path,
+    log: bool = False,
```

Comment:
Just a general question, why would you want the default to ever be to not log?

Reply:
I don't think you would, this is inconsistent also across the codebase (sometimes log=True default), will fix